daycan-front/apps/admin/src/pages/member/components/MemberDataList/MemberDataList.tsx

```diff
@@ -3,12 +3,16 @@ import { MemberDataListHeader } from "../MemberDataListHeader";
 import { MemberDataItems } from "../MemberDataItems/index.ts";
 import { useMember } from "../../hooks/index.ts";
 
-export const MemberDataList = () => {
+interface MemberDataListProps {
+  onEditClick: (memberId: string) => void;
+}
+
+export const MemberDataList = ({ onEditClick }: MemberDataListProps) => {
   const { members } = useMember();
   return (
     <div className={memberDataListContainer}>
       <MemberDataListHeader />
-      <MemberDataItems members={members as any} />
+      <MemberDataItems members={members as any} onEditClick={onEditClick} />
     </div>
   );
 };
```

Comment:
이 부분 구조 개편이 필요해보입니다! 실제 API 와 연동해서 사용하기 위해서는 `MemberDataList`에서부터,` members` 를 받아와야할 것 같아요. 그리고 `MemberDataItems` 라는 파일은 DataList 와 이미 동일한 의미를 갖고 있는 것 같아서 차라리 `MemberDataItem` 으로 이름을 수정하고, 해당 `MemberDataList` 에서 `props` 로 받아온 데이터를 여기서 `map` 함수를 써서 파싱하는 게 훨씬 더 깔끔해보입니다. 지금은 `MemberList` 와 `MemberDataItems` 의 역할이 거의 동일해보여요.
```suggestion
import { MemberDataItems } from "../MemberDataItems/index.ts";
import { useMember } from "../../hooks/index.ts";

interface MemberDataListProps {
  members : memberItem[]; // 예시입니다!
  onEditClick: (memberId: string) => void;
}

export const MemberDataList = ({ onEditClick }: MemberDataListProps) => {
  const { members } = useMember();
  return (
    <div className={memberDataListContainer}>
      <MemberDataListHeader />
      // 여기서 MemberData mapping
      <div className={memberDataList}>
      {members.map((member, idx) => {
     // 이런 식으로

    </div>
  );
};
```